languages.js

```diff
@@ -86,7 +87,8 @@ var instructions = {
     'uk': instructionsUk,
     'vi': instructionsVi,
     'yo': instructionsYo,
-    'zh-Hans': instructionsZhHans
+    'zh-Hans': instructionsZhHans,
+    'fa-IR': instructionsFaIr
```

Comment:
It's better to add your language code in alphabet order not to the the end of the list 😉